prow/config.yaml

```diff
@@ -121,9 +121,9 @@ presubmits:
             path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-cross
-    context: pull-kubernetes-cross
-    rerun_command: "@k8s-bot pull-kubernetes-cross test this"
-    trigger: "@k8s-bot pull-kubernetes-cross test this"
+    context: Jenkins cross build
+    rerun_command: "@k8s-bot cross test this"
```

Comment:
It's a bit of a bicycle shed, but maybe this should also be `@k8s-bot cross build this`? I think that's what it was in the past, but it's different now.